### PR TITLE
Fix modal for Topup balance Generate and mobile menu

### PR DESCRIPTION
**Pull request by MarieTheCoin**

### CODE REVIEW

File: apps/site-app/src/containers/App/styles.module.less [hunk 1]
```diff
@@ -25,7 +25,7 @@
   box-shadow: @subtleShadow;
   
   position: fixed;
-  z-index: 1;
+  z-index: 200;
```
Comment (MarieTheCoin) [inline apps/site-app/src/containers/App/styles.module.less:28]: Puts the app header over everything

File: apps/site-app/src/containers/TopUp/Purchase/eTransferModal.tsx [hunk 2]
```diff
@@ -34,27 +33,28 @@ export const ETransferModal: FunctionComponent<Props> = (props: Props) =>
...
+      closeIcon={true}
+      closeIconFct={props.onCloseDlg}
     >
-      <Grid>
+      <Grid stackable columns={2}>
         <Grid.Row>
           <FormattedMessage {...yourTransferCode} />
         </Grid.Row>
-        <Grid.Row columns="two">
-          <Grid.Column className={styles.fixedColumn}>
```
Comment (MarieTheCoin) [inline apps/site-app/src/containers/TopUp/Purchase/eTransferModal.tsx:44]: Those classes where nowhere in the style file

File: apps/site-app/src/containers/TopUp/Purchase/eTransferModal.tsx [hunk 3]
```diff
@@ -34,27 +33,28 @@ export const ETransferModal: FunctionComponent<Props> = (props: Props) =>
...
+        <Grid.Row>
+          <Grid.Column width="4">
             <FormattedMessage {...yourTransferRecipient} />
           </Grid.Column>
-          <Grid.Column className={styles.floatColumn}>
```
Comment (MarieTheCoin) [inline apps/site-app/src/containers/TopUp/Purchase/eTransferModal.tsx:47]: Those classes where nowhere in the style file

File: apps/site-app/src/containers/App/styles.module.less (hunk 1 above)
Comment (FrozenKiwi) [reply apps/site-app/src/containers/App/styles.module.less:28]: Oh, nice thought!

File: apps/site-app/src/containers/MainNavigation/MainNavigationGreaterThanMobile/index.tsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ import { FormattedMessage } from 'react-intl';
 import HeaderLink from '@thecointech/site-base/components/HeaderLink';
 import { LanguageSwitcher} from '@thecointech/site-base/containers/LanguageSwitcher';
 
-import Logo from './logoAndName.svg';
+import Logo from './images/logoAndName.svg';
```
Comment (FrozenKiwi) [inline apps/site-app/src/containers/MainNavigation/MainNavigationGreaterThanMobile/index.tsx:9]: This seems like a different style than what I've been doing in the past (normally I just drop images beside the component).

I have nothing against the new style - in fact, I can see it being a lot cleaner in a few instances.  If we are making a code-base wide stylistic change though, we should probably start a style guide?

Could you create a style-guide document in the docs folder, and add -just- this bit (images in subfolder)?  I'll make an effort to comform to the new guidelines, and fill out that doc with any further stylistic guidelines if/when making them in future.

File: apps/site-app/src/containers/MainNavigation/MainNavigationMobile/index.tsx [hunk 5]
```diff
@@ -45,7 +45,7 @@ export const MainNavigationMobile = () => {
                   </Menu.Item>
                   <Menu.Item>
                     <Icon name="content" onClick={()=>setModalVisible(true)} />
-                      <ModalOperation isOpen={modalVisible}>
+                      <ModalOperation isOpen={modalVisible} closeIcon={true} closeIconFct={()=>setModalVisible(false)}>
```
Comment (FrozenKiwi) [inline apps/site-app/src/containers/MainNavigation/MainNavigationMobile/index.tsx:48]: just curious - could we merge `closeIcon` into `closeIconFct`?  Can we ever have one without the other?

File: apps/site-app/src/containers/TopUp/Purchase/eTransferModal.tsx (hunk 2 above)
Comment (FrozenKiwi) [reply apps/site-app/src/containers/TopUp/Purchase/eTransferModal.tsx:44]: converting to `less` has broken our styles type-checking.  We need to look into that.  Could you create an issue for that as well?

Comment (FrozenKiwi) [reply apps/site-app/src/containers/TopUp/Purchase/eTransferModal.tsx:44]: The compiler should tell us if we reference non-existent styles.

**Review by FrozenKiwi — APPROVED** [review]
Looks good - please add the additional issues so we don't loose track of them (and the style guide)

File: apps/site-app/src/containers/MainNavigation/MainNavigationGreaterThanMobile/index.tsx (hunk 4 above)
Comment (MarieTheCoin) [reply apps/site-app/src/containers/MainNavigation/MainNavigationGreaterThanMobile/index.tsx:9]: It's already pretty much done everywhere. I will create this doc in another PR.

File: apps/site-app/src/containers/MainNavigation/MainNavigationMobile/index.tsx (hunk 5 above)
Comment (MarieTheCoin) [reply apps/site-app/src/containers/MainNavigation/MainNavigationMobile/index.tsx:48]: Done

**Pull request merged**